zerver/views/realm.py

```diff
@@ -41,6 +42,13 @@ def update_realm(request, user_profile, name=REQ(validator=check_string, default
     if invite_by_admins_only is not None and realm.invite_by_admins_only != invite_by_admins_only:
         do_set_realm_invite_by_admins_only(realm, invite_by_admins_only)
         data['invite_by_admins_only'] = invite_by_admins_only
+    if authentication_methods_enabled is not None and realm.authentication_methods_enabled!= authentication_methods_enabled:
+        if True not in authentication_methods_enabled.values():
+            return json_error(" At least one authentication method must be enabled.",
```

Comment:
Is it intended that there's a leading space in the error message here?  

Also, you should tag the error message for translation with `_()`.